src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/Rss20FeedFormatter.cs

```diff
@@ -275,7 +275,7 @@ private string AsString(DateTimeOffset dateTime)
             }
         }
 
-        private SyndicationLink ReadAlternateLink(XmlReader reader, Uri baseUri)
+        internal static SyndicationLink ReadAlternateLink(XmlReader reader, Uri baseUri, Func<string, UriKind, string, string, Uri> uriParser, bool preserveAttributeExtensions)
```

Comment:
Do we always pass UriParser as uriParser to ReadAlternateLink?

Reply:
No we don't. We have one case where we are calling it from a context where we don't have a user provided parser so we pass in the default parser.